src/ui/components/search/filtersearchcomponent.js

```diff
@@ -98,18 +99,17 @@ export default class FilterSearchComponent extends Component {
     /**
      * The filter string to use for the provided query
      * Optionally provided
+     * TODO(oshi): config.filter is not in the readme. Do we need this?
      * @type {string}
      */
-    this.filter = config.filter || this.core.globalStorage.getState(`${StorageKeys.FILTER}.${this.name}`) || '';
-    if (typeof this.filter === 'string') {
+    this.filter = config.filter || '';
```

Comment:
It seems like the only way the config.filter was used previously was to maybe drive the initial render (via it's usage in setState). But, looking at the Handlebars template of this component, the filter from setState is not actually used anywhere. So, it seems to me like the config.filter option was not actually hooked up to anything. It seems safe to remove, what do you think?